Buildings/Experimental/DHC/Examples/Combined/Generation5/CentralPlants/SewageHeatRecovery.mo

```diff
@@ -0,0 +1,178 @@
+within Buildings.Experimental.DHC.Examples.Combined.Generation5.CentralPlants;
+model SewageHeatRecovery
+  "Model for sewage heat recovery plant"
+  extends DHC.CentralPlants.BaseClasses.PartialPlant(
+    final typ=DHC.Types.DistrictSystemType.CombinedGeneration5,
+    final have_pum=true,
+    have_weaBus=false);
```

Comment:
This base class introduces many parameters that are not used. They should be set to a value with `final` so they disappear in the parameter window.

Reply:
Addressed